service/src/test/java/bio/terra/workspace/service/iam/SamServiceTest.java

```diff
@@ -46,11 +46,7 @@
 import bio.terra.workspace.service.workspace.model.Workspace;
 import bio.terra.workspace.service.workspace.model.WorkspaceStage;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
No `*` imports. You can make sure this doesn't happen with IntelliJ settings. I can show you.